tests/formats/jpeg_test.zig

```diff
@@ -1,7 +1,11 @@
 const helpers = @import("../helpers.zig");
 const jpeg = zigimg.formats.jpeg;
 const std = @import("std");
+const testing = std.testing;
 const zigimg = @import("zigimg");
+const Image = zigimg.Image;
+const color = zigimg.color;
+const utils = @import("../../src/formats/jpeg/utils.zig");
```

Comment:
Don't import files from the implementation in the tests. You can migrate the zig zag test into `utils.zig` directly.